site/test/ColorCalculator.php

```diff
@@ -0,0 +1,79 @@
+<?php
+include('config/init.php');
+
+
+echo "
+	<form action='' method='get'>
+		<select name='Color1'>
+			<option value='Red'>
+			Red
+			</option>
+			<option value='Blue'>
+			Blue
+			</option>
+			<option value='Yellow'>
+			Yellow
+			</option>
+		</select>
+		+
+		<select name='Color2'>
+			<option value='Red'>
+			Red
+			</option>
+			<option value='Blue'>
+			Blue
+			</option>
+			<option value='Yellow'>
+			Yellow
+			</option>
+		</select>
+		<input type='submit' name='submit' />
+	</form>
+";
+
+// var_dump($_REQUEST);
+
+if(isset($_REQUEST['submit'])){
+	if($_REQUEST['Color1'] == 'Red'){
+		if($_REQUEST['Color2'] == 'Red'){
+			echo'<body style="background-color:red">
```

Comment:
I guess this works (who knew?), but it's a little strange looking to me that there isn't a space between `echo` and the string.